parts/after-main.php

```diff
@@ -0,0 +1,64 @@
+<footer class="site-footer">
+ <header><?php echo get_bloginfo( 'name' ); ?></header>
+ <nav class="footer-nav">
+	 <?php
+	 	$spine_site_args = array(
+	 		'theme_location'  => 'site',
```

Comment:
It might be better to register a menu location and use that here instead of using the `menu` argument below.